haskell/src/Ligo/BaseDAO/TZIP16Metadata.hs

```diff
@@ -39,6 +42,16 @@ data MetadataSettings = MetadataSettings
   { msConfig :: MetadataConfig
   }
 
+-- | TZIP-21 assets field.
+data MetadataAsset = MetadataAsset
+  { maName :: Text
+  , maDecimals :: Integer
+  , maDescription :: Text
+  , maShouldPreferSymbol :: Bool
+  , maThumbnailUri :: Text
```

Comment:
since this is optional (and has no default value when unspecified) I believe it should be a
```suggestion
  , maThumbnailUri :: Maybe Text
```
and the field should not appear in the resulting JSON when `Nothing`

Reply:
One more thing: I think that this should be given as an option for users to specify if they want to.

Reply:
I've make it configurable now, also updated the makefile as well.